src/api/request.ts

```diff
@@ -32,7 +32,7 @@ export interface IFetch {
    */
   stringifyBody?: boolean;
 }
-let apiURL = process.env.KAOTO_API;
+let apiURL = 'http://localhost:8081';
```

Comment:
Gotta figure out how to use `process.env` when building directly with Typescript.